protovalidate/internal/extra_func.py

```diff
@@ -548,6 +548,8 @@ def get_bits(self) -> int:
 
         # Handle double colon, fill pieces with 0
         if self._double_colon_seen:
+            if len(p16) >= 7:
+                return 0
```

Comment:
Why the return of `0`?

Reply:
This is because `get_bits` is supposed to return `0` if the address is invalid.

```python
    def get_bits(self) -> int:
        """Return the 128-bit value of an address parsed through address() or address_prefix().

        Return 0 if no address was parsed successfully.

        """
```
OTOH, though, it doesn't really matter, so I removed it.